packages/cli/src/services/translationIO.ts

```diff
@@ -14,8 +15,16 @@ const getCreateHeaders = (language) => ({
   Language: language,
 })
 
+const getTargetLocales = (config: LinguiConfig) => {
```

Comment:
Please use `LinguiConfigNormalized` type here, and in all other cases in this file

Reply:
Done